teuthology/task/ceph_ansible.py

```diff
@@ -1,15 +1,22 @@
 import json
 import os
+import re
+import logging
+import yaml
 
 from cStringIO import StringIO
 
-from . import ansible
-
+from . import Task
+from tempfile import NamedTemporaryFile
 from ..config import config as teuth_config
 from ..misc import get_scratch_devices
+from teuthology import contextutil
+from teuthology.orchestra import run
+from teuthology import misc as teuthology
```

Comment:
why import misc as teuthology?  Seems like you could just ``from teuthology import misc`` and it'd make more sense.